ramips: use lzma-loader for Wevo devices

orangepizza opened the pull request:
As kernel size increased it start to fail to load squishfs image, using lzma-loader fixed it
wevo_11acnas is almost same device as w2914ns-v2 except ram size, so I expect same thing would've happen in that device too.

Signed-off-by: Seo Suchan <abnoeh@mail.com>

mans0n commented:
1. The title should be `ramips: use lzma-loader for WeVO devices`.
2. SoB uses `<>` instead of `()`.
3. The [From header](https://github.com/openwrt/openwrt/commit/6718526dcb91a4bd68d85770d4185a8aa0bc9f2d.patch) should match with the SoB. You can do this with `git config -e` and `git commit --amend --no-edit --reset-author`.
4. Please wrap long commit message lines. (< 75 characters per line)

orangepizza commented:
I changed mail to the one github uses.

mans0n commented:
Please use lowercase `ramips` in your commit message as well.
And the name in the From header still does not match the name in the SoB.

orangepizza commented:
does it works now?

mans0n commented:
Please try:
```sh
# check
git -P show -s --format='Author: %an <%ae>'

# fix
git config user.name <username>
git config user.email <useremail>
git commit --amend --no-edit --reset-author

# double check
git -P show -s --format='Author: %an <%ae>'
```

orangepizza commented:
ah I show the wrong mail on github profile. I think that domain is expired. I expect change every to abnoeh@mail.com currently.

mans0n commented:
I mean, your mail address is fine, but your patch still uses github username, `orangepizza`. You need to fix that.
`From: orangepizza <abnoeh@mail.com>`
My comments were not specific enough, sorry for the confusion.

orangepizza committed:
ramips: use lzma-loader for Wevo devices

As kernel size increased it start to fail to load squishfs image,
using lzma-loader fixed it.
wevo_11acnas is almost same device as w2914ns-v2 except ram size,
so I expect same thing would've happen in that device too.

Signed-off-by: Seo Suchan <abnoeh@mail.com>

mans0n commented:
LGTM now. Thanks for your patience with my nitpicking.

orangepizza commented:
if possible can you land this patch to 21.02? it will failed to load in default setting.

adschm commented:
@mans0n Reviewed-by?

mans0n commented:
`Reviewed-by: Sungbo Eo <mans0n@gorani.run>`

adschm commented:
Thanks, will be merged later today.

BTW: mt7621_zio_freezio.dts uses the same mt7621_wevo_w2914ns-v2.dtsi, so maybe it should be dealt with as well?

mans0n commented:
Hmm good point, their boards are almost identical, but they use different firmware so I can't be sure about that without a test.
